lnwallet/channel_test.go

```diff
@@ -4802,20 +4818,26 @@ func TestChanAvailableBalanceNearHtlcFee(t *testing.T) {
 
 		htlcIndex++
 		aliceBalance -= htlcAmt
+		bobBalance += htlcAmt
 	}
 
 	// Balance should start out equal to half the channel capacity minus
 	// the commitment fee Alice must pay and the channel reserve. In
 	// addition the HTLC fee will be subtracted fromt the balance to
 	// reflect that this value must be reserved for any payment above the
 	// dust limit.
-	expBalance := aliceBalance - commitFee - aliceReserve - htlcFee
-	checkBalance(t, expBalance)
+	expAliceBalance := aliceBalance - commitFee - aliceReserve - htlcFee
```

Comment:
name this `expAliceBalance` already in the earlier commit, to make the delta smaller?

Reply:
Done: https://github.com/lightningnetwork/lnd/pull/3691/commits/1354a838e213c89b73cdc288414d87c3b02843e9